src/sql/workbench/parts/notebook/outputs/notebookMarkdown.ts

```diff
@@ -0,0 +1,240 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as path from 'path';
+
+import { URI } from 'vs/base/common/uri';
+
+import { IDisposable, dispose } from 'vs/base/common/lifecycle';
+import { RenderOptions } from 'vs/base/browser/htmlContentRenderer';
+import { IMarkdownString, removeMarkdownEscapes } from 'vs/base/common/htmlContent';
+import { IMarkdownRenderResult } from 'vs/editor/contrib/markdown/markdownRenderer';
+import marked = require('vs/base/common/marked/marked');
+import { defaultGenerator } from 'vs/base/common/idGenerator';
+import { revive } from 'vs/base/common/marshalling';
+import * as fs from 'fs';
+
+export class NotebookMarkdownRenderer {
+	private _notebookURI: URI;
+	private _baseUrls: string[] = [];
+
+	constructor() {
+	}
+
+	render(markdown: IMarkdownString): IMarkdownRenderResult {
+		let disposables: IDisposable[] = [];
```

Comment:
Do we intent to use this disposable in the future?

Reply:
Unsure. I can clean this up some more.